manim/mobject/types/vectorized_mobject.py

```diff
@@ -1029,6 +1029,11 @@ def __repr__(self):
             + ")"
         )
 
+    def __str__(self):
+        return (
+            f"self.__class__.__name__({len(self.submobjects)})"
```

Comment:
Currently, this produces output like:

```py
Animation 1: Transformself.__class__.__name__(484):   0%|          | 0/30 [00:00<?, ?it/s]
Animation 1: Transformself.__class__.__name__(484):   3%|▎         | 1/30 [00:00<00:16,  1.75it/s]
Animation 1: Transformself.__class__.__name__(484):   7%|▋         | 2/30 [00:01<00:15,  1.80it/s]
```

so I don't think we are there yet.

```suggestion
            f"{self.__class__.__name__} of {len(self.submobjects)} submobjects"
```

Reply:
I don't have access to a laptop so I was mostly guessing on those format strings.

If you could kindly also help me out with the black accepted format for your fix